### PR TITLE
Updated Bower command in case permission errors happen

### PR DESCRIPTION
**Pull request by himagirim**

### CODE REVIEW

**Comment by himagirim** [comment]
Error!!

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ If you don't have them already, you'll need node, bower and gulp to be installed
 Change directory into the new project you just cloned, then install dependencies.
 ```
 npm install
-bower install
+bower install (Use `sudo bower install --allow-root` if you see permission issues")
```
Comment (sks) [inline README.md]: or you should change the folder permission

```
sudo chown -R `whoami` .
```

**Comment by turnerth** [comment]
Thanks Sabith.  To be more specific, we don't recommend using sudo to run these commands.  It can cause lots of problems later.  e.g. sudo launches another shell and there can be problems with environment variables, particularly http_proxy and https_proxy.  

Make sure you are running the command in the predix-seed directory in your user home directory.  This should not have the permissions problems you are seeing.

**Comment by himagirim** [comment]
I did it in my user home directory and I still see that error. Canceled the pull request for now.

**Comment by gedesigntech** [comment]
bower sometimes has these permissions hiccups.  will look more into it

On Feb 16, 2017, at 5:32 PM, Himagiri (Hima) Mukkamala <notifications@github.com<mailto:notifications@github.com>> wrote:


I did it in my user home directory and I still see that error. Canceled the pull request for now.

**Pull request closed without merging**